gatling/user-files/utils/delete-users.js

```diff
@@ -20,10 +20,9 @@ const doit = async (id) => {
 };
 const delete_users = async () => {
   try {
-    let id = 1;
-    for (let it = 1; it <= 20; ++it) {
-      await doit(id);
-      id += 100;
+    const usersBundleSize = 100;
```

Comment:
This is still (somewhat) a magic number, even if it is better defined, because the exact same number must be used in different parts of the program for it to work, and you defined different constants (with the same value) at different places.

Whenever you need some kind of fixed value, the best practice is to define once and for all a unique constant, with a scope greater than the components needing it, so that it can be accessed by those. 
That practice's goal is that, whenever you need to change a fixed value, you only have to do it once, at only one place, without fear of side effects.
As an added benefit, this practice allows, if needed (I'm not sure that it's the case here), to add a descriptive comment to it, to help future maintainers understanding what the value is for.
Here is an example: https://github.com/kuzzleio/kuzzle/blob/master/lib/api/core/auth/tokenManager.js#L29

You should declare that variable as a global constant, at the beginning of that program.

Reply:
solved with a `config.json` file :)